src/v/config/configuration.cc

```diff
@@ -1221,15 +1221,15 @@ configuration::configuration()
       "kafka_nodelete_topics",
       "Prevents the topics in the list from being deleted via the kafka api",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
-      {"__audit", "__consumer_offsets", "_schemas"},
+      {"__audit_log", "__consumer_offsets", "_schemas"},
```

Comment:
I personally think having this in the list is a good thing.  Would make the customer think twice before deleting the topic when auditing is disabled.

Reply:
It's just another layer of protection (when auditing is disabled) to protect the log from inadvertent deletion.  As this cluster config is modifiable the customer can remove it from the list if desired (like they would do with _schemas if necessary).

Reply:
fair point!  let's defer to someone with decision making "authority"

Reply:
Given that customers just need to do an `rpk cluster config set ...` command to remove it, I think the guardrail would be appreciative, but this will require documentation to call out how to delete the audit topic.